### ISSUE TITLE
Trying to figure this out with Portainer and an external NPM

### ISSUE THREAD

**Issue by quadcom** [xBrowserSync/api-docker#35]
While I appreciate the compose YAML, I'm having a frustrating time adapting the example to use my NPM running on my Unraid server while setting this up in Portainer on one of my other servers.

I'm not the most experienced with either Docker or Compose, for that matter, so everything I've tried so far hasn't worked.

I'd really appreciate a little guidance on how to spin this up in this environment.

Thx

**Comment by pamart** [comment]
I agree, a guide to simply deploy xbrowsersync without a built-in reverse proxy would be great!
I gave up using this app because of that :(

**Comment by ArticExploit** [comment]
just remove the caddy container from the docker-compose.yml, from there just follow the same steps as any other service to run through the reverse proxy you're already using
